deploy/helm/scf/assets/operations/instance_groups/api.yaml

```diff
@@ -210,18 +275,26 @@
       healthcheck:
         file_server:
           readiness:
-            tcpSocket:
+            httpGet:
+              path: /v1/static/file_server/bin/file-server
               port: *file-server-port
+          liveness:
+            exec:
+              command: [pidof, file-server]
 
 # Add quarks properties for statsd_injector.
 - type: replace
-  path: /instance_groups/name=api/jobs/name=statsd_injector/properties/quarks?
+  path: /instance_groups/name=api/jobs/name=statsd_injector/properties/quarks?/run/healthcheck/statsd_injector
   value:
-    ports:
-    # TODO: Can we remove this port?
```

Comment:
Yes, we could. It never worked anyway, since statsd-injector was listening on **UDP** port 8125.